lib/redis-time-series.rb

```diff
@@ -1,6 +1,5 @@
 require 'bigdecimal'
 require 'forwardable'
-require 'ext/time_msec'
```

Comment:
I never particularly cared for this refinement, but curious if y'all ran in to issues with it or just also disliked the implementation?